policyengine_us/parameters/gov/states/hi/tax/income/rates/single.yaml

```diff
@@ -25,11 +25,11 @@ brackets:
     rate:
       2017-01-01: 0.072
   - threshold:
-      2017-01-01: 24_000
+      2017-01-01: 36_000
```

Comment:
this change is on purpose right?

Reply:
I remember I added 24000 here... According to the merged version, it should be 24000.